### PR TITLE
Release 0.4.0

### PR DESCRIPTION
**Pull request by razvan**
Release 0.4.0. DO NOT SQUASH MERGE!

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ description = "Stackable Operator Framework"
 edition = "2018"
 license = "Apache-2.0"
 name = "stackable-operator"
-version = "0.4.0-nightly"
+version = "0.5.0-nightly"
```
Comment (fhennig) [inline Cargo.toml:7]: Why did you bump the version here?
If that's required, it should also be in the changelog

Comment (razvan) [reply Cargo.toml:7]: No, this is the _next development version_. In the CHANGELOG the `[Unreleased]` line will be replaced with 0.5.0 the next time we make a release.

Comment (fhennig) [reply Cargo.toml:7]: Alright :+1:

**Review by fhennig — APPROVED** [review]
good :+1:

**Pull request merged**